lasagne/objectives.py

```diff
@@ -338,6 +338,50 @@ def multiclass_hinge_loss(predictions, targets, delta=1):
     return theano.tensor.nnet.relu(rest - corrects + delta)
 
 
+def huber_loss(predictions, targets, delta=1):
+    """ Computes the huber loss between predictions and targets.
+
+    .. math:: L_\delta (diff) = \frac{diff^2}{2} & \text{if  } |diff| \le \
+                                \delta
+    .. math:: L_\delta (diff) & = & \delta (|diff| - \frac{\delta}{2} ),\
+                                     &\text{else}
```

Comment:
I used two here because there are two different expressions that this will return, based on the value of delta. To represent that, I used two math.